apps/web/pages/settings/my-account/out-of-office/index.tsx

```diff
@@ -86,26 +115,71 @@ const CreateOutOfOfficeEntryModal = ({
     },
   });
 
+  const resetForm = useCallback(() => {
+    setValue("toTeamUserId", null);
+    setValue("notes", "");
+    setValue("dateRange", dateRange);
+    setValue("reasonId", 1);
+    setValue("uuid", null);
+    setSelectedReason(null);
+    setSelectedMember(null);
+    setProfileRedirect(false);
+  }, [dateRange, setValue]);
```

Comment:
Hey, why are you doing extra work? Can't we just use the `reset` method of `useForm` and call it?

Reply:
Done!

But one question - I have kept the `resetForm` function to reset the state values, i.e `selectedMember`, etc.

Although I don't see an use of that, because the form fields only acts as the state, but as the states were present therefore I did reset them also.